content/working_with/console/pages/dashboard-page.md

```diff
@@ -14,7 +14,7 @@ The Dashboard page is the landing page when you log into the {{< param cfy_conso
 
 ### Statistics
 
-By default, the page displays usage statistics of the {{< param cfy_manager_name >}}, including number of services and number of Kubernetes clusters on this {{< param cfy_manager_name >}}.
+By default, the page displays usage statistics of the {{< param cfy_manager_name >}}, including number of: services, Kubernetes clusters and Terraform modules on this {{< param cfy_manager_name >}}.<br />
```

Comment:
Oh yea, I think that at first it I had idea about introducing series of mehods
But as I'm looking at it right now it may be redundant 😁 

Thanks for pointing that out!

Reply:
Colon has been removed 🎉